Dockerfile_custom

```diff
@@ -0,0 +1 @@
+FROM postgis/postgis:16-3.4
```

Comment:
Ideally, we need to create our own Dockerfile, which could start from the postgis image we're currently using and then add the pgvector extension.